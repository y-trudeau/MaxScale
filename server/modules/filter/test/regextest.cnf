[Regex]
type=filter
module=regexfilter
match=wrong
replace=right
