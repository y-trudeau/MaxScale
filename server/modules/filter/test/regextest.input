select * from mysql.wrong;
select wrong from test.table;
select * from test.table where name='wrong';