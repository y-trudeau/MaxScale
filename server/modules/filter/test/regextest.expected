select * from mysql.right;
select right from test.table;
select * from test.table where name='right';
